### PR TITLE
Change file names and update imports

### PR DESCRIPTION
**Pull request by NFartenadze**

### CODE REVIEW

**Comment by NFartenadze** [comment]
after renaming folders, imports were changed automatically, after restarting vs code, those imports showed error so i fixed them again

File: src/index.ts [hunk 1]
```diff
@@ -4,7 +4,9 @@ import * as dotenv from "dotenv";
 dotenv.config();
 
 mongoose
-  .connect(process.env.MONGODB_URI!)
+  .connect(
+    "mongodb+srv://farnick1148:o1vvfQtCYh4ehKJK@cluster0.d9r34ci.mongodb.net/banking-system/?retryWrites=true&w=majority"
```
Comment (FCosta20) [inline src/index.ts]: Please don't hardcode credentials

**Review by FCosta20 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (NFartenadze) [reply src/index.ts]: okay, i had them in .env, but i was told to remove .env, so i hardcoded it for you to use it, since it's not real-life application.

Comment (FCosta20) [reply src/index.ts]: Yeah, it's not a real app but it's not a good practice to push credentials into remote repo

**Review by FCosta20 — APPROVED** [review]

**Pull request merged**